wild_lib/src/resolution.rs

```diff
@@ -769,7 +772,10 @@ fn resolve_sections_for_object<'data>(
                             SectionSlot::Unloaded(UnloadedSection::new(id))
                         }
                         TemporaryPartId::Custom(custom_section_id, _alignment) => {
-                            if custom_section_id.name.bytes().starts_with(b".debug_") {
+                            let section_name = custom_section_id.name.bytes();
+                            if section_name == b".note.gnu.property" {
```

Comment:
All right, let me use `TemporaryPartId::NoteGnuProperty`, hope it's the right approach?!

Reply:
Rather than introducing a new variant of `TemporaryPartId`, would it work to use `TemporaryPartId::BuiltIn(part_id::NOTE_GNU_PROPERTY:)`?

Reply:
I'll wait in case you wanted to try out `BuiltIn`. If you decide you're happy with it as-is, let me know and I'll merge

Reply:
Yes, I would like to adjust the code so that it uses `TemporaryPartId::BuiltIn`, I will go back to that over the weekend. Plus, I would like to integrate a linker-diff pass for this section - as mentioned the bit-wise comparison is not enough!

Reply:
If I use `TemporaryPartId::BuiltIn(part_id::NOTE_GNU_PROPERTY:)` I get crash here:
```
thread '<unnamed>' panicked at wild_lib/src/output_section_id.rs:554:13:
part_id_with_alignment can only be called for regular sections
```

that's called here:
https://github.com/davidlattimore/wild/blob/c8bbc8df13ef5bb23d8bf6fd49cbb3922704d86a/wild_lib/src/part_id.rs#L184

Any thought on that?

Reply:
One option would be to construct the return value directly after you check the section name. e.g.

```rust
if section_name == b".note.gnu.property" {
    return Ok(Some(UnresolvedSection {
        part_id: TemporaryPartId::BuiltIn(part_id::NOTE_GNU_PROPERTY),
        is_string_merge: false
    }));
}
```

Reply:
Sure, that works, thanks.